src/algorithms/crypto/randomness.md

```diff
@@ -10,9 +10,10 @@ We describe this formatting below.
 
 ## Encoding On-chain data for randomness
 
-Any randomness derived from on-chain values uses the following encodings to represent these values as bytes
+Any randomness derived from on-chain values uses the following encodings to represent these values as bytes:
+
 - **Bytes**: Bytes
-- **Ints**: Little endian uint64 representation
+- **Ints**: Little-endian uint64 representation
```

Comment:
As discussed [here](https://filecoinproject.slack.com/archives/CHMNDCK9P/p1576707533036200), CBOR encodes ints big-endian, so specifiying little-endian here will require re-serialization which is pretty low value.

Reply:
Happy to do big-endian, note though that this is encoding for the purposes of randomness generation, so it should not get CBOR serialized thereafter...